### PR TITLE
misc: fix docs accessibility

### PR DESCRIPTION
**Pull request by lauzadis**
## Description of changes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/dokka-presets/css/aws-styles.css [hunk 1]
```diff
@@ -56,6 +56,10 @@
     overflow-wrap: break-word;
 }
 
+.keyValue {
```
Comment (0marperez) [inline docs/dokka-presets/css/aws-styles.css:59]: question: where is this used?

File: docs/dokka-presets/css/aws-styles.css [hunk 2]
```diff
@@ -56,6 +56,10 @@
...
+    display: block;
+}
+
```
Comment (0marperez) [inline docs/dokka-presets/css/aws-styles.css:62]: question: where is this used?

Comment (lauzadis) [reply docs/dokka-presets/css/aws-styles.css:62]: It overrides a style defined by Dokka: https://github.com/Kotlin/dokka/blob/8f386ba7aa68620376e56b87936263335da7f116/dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css#L1283-L1286

**Review by 0marperez — APPROVED** [review]

File: docs/dokka-presets/css/aws-styles.css (hunk 2 above)
Comment (0marperez) [reply docs/dokka-presets/css/aws-styles.css:62]: Should it be in [this PR](https://github.com/smithy-lang/smithy-kotlin/pull/1132) as well?

Comment (lauzadis) [reply docs/dokka-presets/css/aws-styles.css:62]: Yes, thanks for catching it

**Review by ianbotsf — APPROVED** [review]

**Pull request merged**